reports/Spatial Outliers.qmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "Distributions `r params$run_id`"
+execute:
+  echo: false
+  warning: false
+format:
+  html:
+    embed-resources: true
+    toc: true
+    toc_float: true
+    fig-align: center
+    fontsize: 12pt
+editor: source
+params:
+  run_id: 2023-03-14-clever-damani
+  year: '2023'
+---
+
+```{r}
+library(knitr)
+source("Setup_File.R")
+```
+
+## Standardized mean differences
+```{r outlier_map, message=FALSE}
+# This should be replaced with
+outliers <- read_parquet("~/Separate Quarto doc into file per topic #74/reports/Separate Quarto doc into file per topic #74/output/test_card/model_test_card.parquet")
+
+
+# that would be trimmed according to CCAO SOPs using the is_outlier() function
+outlier_dist <- outliers %>%
+  group_by(meta_township_code) %>%
```

Comment:
Just want to flag that it was in the original code, but don't know the relevance for the updated setup. No grouping would allow for percentages by township to be calculated.